<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeComponent</class>
 <widget class="QWidget" name="VolumeComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>体数据</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>当前操作体数据编号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_currVolID">
          <item>
           <property name="text">
            <string>0号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1号</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>RAW体数据</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_voxPerVolX">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1024</number>
           </property>
           <property name="value">
            <number>300</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spinBox_voxPerVolY">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1024</number>
           </property>
           <property name="value">
            <number>350</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="4">
          <widget class="QPushButton" name="pushButton_LoadRAWVolume">
           <property name="text">
            <string>加载RAW数据</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>尺寸(X,Y,Z)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="spinBox_voxPerVolZ">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1024</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>体素类型 UInt8</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>传输函数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_LoadTF">
           <property name="text">
            <string>加载传输函数</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
